### PR TITLE
listen for page being ready as well as menu

### PR DESCRIPTION
**Pull request by moloko**
fixes https://github.com/adaptlearning/adapt_framework/issues/3129

### CODE REVIEW

**Comment by moloko** [comment]
Just to note that adapt-contrib-bookmarking [also listens to both events](https://github.com/adaptlearning/adapt-contrib-bookmarking/blob/ffb4ac92d632439118806b2b57f545c84fd2e3ca/js/adapt-contrib-bookmarking.js#L39)

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by eleanor-heath — APPROVED** [review]

File: js/languagePickerModel.js [hunk 1]
```diff
@@ -102,7 +104,7 @@ define([
       if (!this.get('_restoreStateOnLanguageChange')) {
         return;
       }
-      this.listenToOnce(Adapt, 'menuView:ready', this.restoreLocation);
+      this.listenTo(Adapt, 'pageView:ready menuView:ready', this.restoreLocation);
```
Comment (oliverfoster) [inline js/languagePickerModel.js:107]:
```suggestion
      this.listenTo(Adapt, 'contentObjectView:ready', this.restoreLocation);
```
https://github.com/adaptlearning/adapt_framework/blob/master/src/core/js/views/contentObjectView.js#L71

**Review by oliverfoster — APPROVED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by ethan-lp — APPROVED** [review]
👀

**Pull request merged**